src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -87,6 +92,16 @@ const IntegrationRow = (props: Props) => {
           ))}
         </InternalContainer>
       </FlexContainer>
+      {alertText && (
+        <AlertContainer>
+          <Alert type="warning" icon={<IconWarning size="sm" />}>
```

Comment:
I thought we wanted to have the alerts be of type `error`

Reply:
@MeredithAnya Nah, there was a large debate in #discuss-design and we decided to go with warning.